docs/manual/faq/_index.en.md

```diff
@@ -1,111 +1,133 @@
+{{% expand "How can I use an insert tag in my template?" %}}
+To use an [Insert-Tag](/en/article-management/insert-tags/) `{{date}}` in your template you have to insert 
+`$this->replaceInsertTags('{{date}}')`.
 {{% /expand %}}
 
-{{% expand "Wie kann ich den TinyMCE-Editor konfigurieren?" %}}
-The information about this can be found under [TinyMCE Editor Configuration](/de/anleitungen/tinymce-konfiguration/).
+{{% expand "How can I configure the TinyMCE editor?" %}}
+The information about this can be found under [TinyMCE Editor Configuration](/en/guides/tinymce-configuration/).
 {{% /expand %}}
 
+
 ## Configuration and adjustment
 
-{{% expand "Es wird keine E-Mail über mein Formular versendet, was muss ich machen?" %}}
-Check in or add `parameters.yml` the [SMTP details](/de/system/einstellungen/#smtp-versand) of your hoster Then you have to empty the application cache once via Contao Manager ("System maintenance" &gt; "Replace product cache") or via the console.
+{{% expand "No e-mail is sent via my Form, what do I have to do?" %}}
+Check in or add `parameters.yml` the [SMTP details](/en/system/settings/#e-mail-sending-configuration) of your hoster. 
+Then you have to empty the application cache once via Contao Manager ("System maintenance" &gt; "Replace product cache") 
+or via the console.
 {{% /expand %}}
 
-{{% expand "Kann ich eine andere E-Mail-Adresse als Absender für Formulare benutzen?" %}}
-The e-mail address set in the "Settings &gt; System administrator's e-mail address" area is also used as the sender of forms. You can enter an additional e-mail address in the "Page structure" section for the "Starting point of a website" page type. This e-mail address will then be used as the sender.
+{{% expand "Can I use a different e-mail Address as sender for forms?" %}}
+The e-mail address set in the "Settings &gt; System administrator's e-mail address" area is also used as the sender of forms. 
+You can enter an additional e-mail address in the "Page structure" section for the "Starting point of a website" page type. 
+This E-Mail address will then be used as the sender.
+Since Contao 4.10 you can [configure](/en/system/settings/#different-e-mail-configurations-and-sender-addresses) this.
 {{% /expand %}}
 
-{{% expand "Wie kann ich das Sprachkürzel der URL hinzufügen?" %}}
-You can add the entry `prepend_locale: true` in [config.yml](/de/system/einstellungen/#config-yml) and then you have to empty the application cache once using the Contao Manager ("System Maintenance" &gt; "Replace Prod. cache") or the console.
+{{% expand "How can I add the language abbreviation to the URL?" %}}
+You can add the entry `prepend_locale: true` in [config.yml](/en/system/settings/#config-yml) and then you have to 
+empty the application cache once using the Contao Manager ("System Maintenance" &gt; "Replace Prod. cache") or the console.
```

Comment:
```suggestion
empty the application cache once using the Contao Manager ("Maintenance" &gt; "Application Cache" &gt; "Rebuild Production Cache") or the console.
Since Contao 4.10, you can freely define the URL prefix, independently of the language.
```